packages/hooks/src/useVerifyEmail.ts

```diff
@@ -34,6 +34,7 @@ const useVerifyEmail = (
 
     return {
         is_loading: WS.isLoading,
+        is_success: WS.isSuccess,
```

Comment:
May be we can just spread WS. That way we get all status and no need to override the names of the props